com.teamspeak.TeamSpeak3.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: com.teamspeak.TeamSpeak3
+tags:
+  - proprietary
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: teamspeak3
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.ts3client
+  - --filesystem=xdg-download
+  - --env=SSL_CERT_DIR=/etc/ssl/certs
+modules:
+  - name: libevent
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+  - name: teamspeak3
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply_extra /app/bin/apply_extra
+      - install -Dm755 teamspeak3 /app/bin/teamspeak3
+      - install -Dm644 -t /app/share/metainfo com.teamspeak.TeamSpeak3.appdata.xml
+      - install -Dm644 -t /app/share/applications com.teamspeak.TeamSpeak3.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/128x128/apps com.teamspeak.TeamSpeak3.png
+    sources:
+      - type: file
+        path: com.teamspeak.TeamSpeak3.desktop
+      - type: file
+        path: com.teamspeak.TeamSpeak3.png
+      - type: file
+        path: com.teamspeak.TeamSpeak3.appdata.xml
+      - type: extra-data
+        only-arches:
+          - x86_64
+        filename: TeamSpeak3-Client.run
+        url: https://files.teamspeak-services.com/releases/client/3.6.2/TeamSpeak3-Client-linux_amd64-3.6.2.run
+        sha256: 59f110438971a23f904a700e7dd0a811cf99d4e6b975ba3aa45962d43b006422
+        size: 110899830
+        x-checker-data:
+          type: html
+          url: https://files.teamspeak-services.com/releases/client/
+          version-pattern: (\d+\.\d+\.\d+)
```

Comment:
are you sure this won't grab 4.x version?

Reply:
There is no 4.x version, they skipped to 5.0. I am not sure, but I think the whole check is obsolete (the url is just returning 404)

Reply:
I implemented a new update checker, but it depends on https://github.com/flathub-infra/flatpak-external-data-checker/pull/410

Reply:
are you sure this won't grab 5.x version?


since you want to be pedantic, I rephrased the question.

Reply:
As I said:
> I am not sure, but I think the whole check is obsolete (the url is just returning 404)

Its broken, I have a local check committed but it depends on the said pull request (its binary data) and does not work with the current yaml format.

Reply:
if the check doesn't work why is it here?

Reply:
Because it was and I did not bother removing it.